### PR TITLE
Format code with Gleam v1.1

### PR DESCRIPTION
**Pull request by maxdeviant**
This PR updates the version of Gleam used to v1.1 and runs the formatter.

### CODE REVIEW

File: src/birl.gleam [hunk 1]
```diff
@@ -300,7 +300,8 @@ pub fn parse(value: String) -> Result(Time, Nil) {
   {
     [day_string, time_string], _, _
     | _, [day_string, time_string], _
-    | _, _, [day_string, time_string] -> Ok(#(day_string, time_string))
+    | _, _, [day_string, time_string]
+    -> Ok(#(day_string, time_string))
```
Comment (maxdeviant) [inline src/birl.gleam:304]: @giacomocavalieri Does this formatting look right to you? Or is this a bug in the formatter?

Comment (giacomocavalieri) [reply src/birl.gleam:304]: Mmh this look like a bug, could you please open an issue on the gleam repo? (Maybe tag me there too, so I don't forget about this 😁)

Comment (maxdeviant) [reply src/birl.gleam:304]: Opened as https://github.com/gleam-lang/gleam/issues/3136.

**Comment by massivefermion** [comment]
Thanks for the PR and happy this caused an issue to be recognized. But since I'd probably need to change some stuff and I also want to arrange the inputs the way I like it, I'm not going to merge this PR.

**Pull request closed without merging**